docs/coding/two-phase-transfers.md

```diff
@@ -31,12 +31,22 @@ A post-pending transfer, denoted by
 pending transfer to "post", transferring some or all of the pending transfer's reserved amount to
 its destination.
 
+- If the posted [`amount`](../reference/transfer.md#amount) is greater than or equal to the pending
+  transfer's amount, the full pending transfer's amount is posted.
```

Comment:
Explain that if the amount is greater than the amount of the pending transfer, the value recorded is set to the original amount.